### DISCUSSION TITLE
Unable to start the software

### DISCUSSION THREAD

**Discussion by pw310** [aaronwmorris/indi-allsky#1393] [Q&A] [answered]
HI, I need some help please...

Flashed Bookworm build 2023.11.09 to SD using Balena Etcher
Inserted SD card into Raspberry Pi 4 (4GB) and powered on
Pi already has a Raspberry Pi Camera Module 3 (Wide Angle) attached (Sony IMX708 sensor)

After the initial setup I followed the getting started routine...

When I came to start the software, I get the following

allsky@allsky:~/indi-allsky $ sudo systemctl --user start indi-allsky
Failed to connect to bus: No medium found

I have tried with the Raspberry Pi imager and Balena Etcher

I'm using the 64 bit Bookworm without the desktop

I don't know very much about linux, so a bit lost here...

**Comment by aaronwmorris** [comment]
When you run the systemctl command, run it without sudo

**Reply by pw310** [reply]
I tried that initially and get this...

allsky@allsky:~ $ systemctl --user start indi-allsky
Failed to start indi-allsky.service: Unit indi-allsky.service not found.

**Reply by aaronwmorris** [reply]
It looks like the setup.sh script did not fully complete

**Reply by pw310** [reply, marked as answer]
ahh, I believe I need to run setup.sh again, on the first attempt I got..

Please run ./misc/build_indi.sh before running setup.sh

I ran that, but didn't run the setup again!
